pkg/deploy/service.go

```diff
@@ -62,7 +63,20 @@ func CreateDeploymentRequest(application string, config *model.OrchestrationConf
 				},
 			},
 			Constraints: &pipelineConstraint,
-		})
+		}
+		if config.Analysis != nil {
+			if config.Analysis.DefaultAccount == "" {
+				return nil, fmt.Errorf("analysis configuration block is present but default account not set")
+			}
+			if config.Analysis.DefaultType == "" {
+				return nil, fmt.Errorf("analysis configuration block is present but default type not set")
+			}
+			analysis.DefaultAccount = &config.Analysis.DefaultAccount
+			analysis.DefaultType = &config.Analysis.DefaultType
+			analysis.Queries = CreateAnalysisQueries(*config.Analysis.Queries, config.Analysis.DefaultAccount)
+			deploymentToAdd.Analysis = &analysis
+		}
```

Comment:
Doesn't this block evaluate to the same object every run? This should be create once, outside of the Targets loop, and added to each deployment.

Reply:
good catch, I'll update in a bit.